tests/package/test_context_handler.py

```diff
@@ -107,3 +107,10 @@ def test_custom_packagers(
         assert False
     except RunError:
         pass
+
+
+def test_is_logging_worker(patch_mpi4py):
```

Comment:
I like to add doc strings to tests as well. You won't find a single test / attribute / method without one in modules I'm in charge of :)

So, please add something like:
```suggestion
def test_is_logging_worker(patch_mpi4py):
"""
Test the `_is_logging_worker` method of the context handler to return True for when the is an `mpi4py` mock. 

    :param patch_mpi4py: A `mpi4py` module mock fixture.
"""
```